docs/filtering.md

```diff
@@ -0,0 +1,324 @@
+# Filtering
+
+One of the easiest way to lower your ingest is to filter out data you do not need. In this guide you will learn how to do this for logs,
+metrics and their metadata.
+
+- [OpenTelemetry Collector processors](#opentelemetry-collector-processors)
+  - [Filter processor](#filter-processor)
+  - [Transform processor](#transform-processor)
+- [Metadata](#metadata)
+  - [Removing unnecessary metadata](#removing-unnecessary-metadata)
+  - [Truncating too long attributes](#truncating-too-long-attributes)
+  - [Replacing long substring in attributes](#replacing-long-substrings-in-attributes)
+- [Logs](#logs)
+  - [Excluding logs from specific sources](#excluding-logs-from-specific-sources)
+  - [Custom filtering](#custom-filtering)
+    - [Shorten log body](#shorten-log-body)
+    - [Drop unnecessary logs](#drop-unnecessary-logs)
+- [Metrics](#metrics)
+  - [Prometheus](#prometheus)
+    - [Filtering Prometheus metrics by namespace](#filtering-prometheus-metrics-by-namespace)
+  - [Filter out app metrics](#filter-out-app-metrics)
+  - [Custom filtering](#custom-filtering-1)
+    - [Drop unnecessary metrics](#drop-unnecessary-metrics)
+
+## OpenTelemetry Collector processors
+
+Basic way to perform any custom filtering is adding additional processors to the pipeline in the metadata layer.
```

Comment:
```suggestion
This Helm Chart uses the OpenTelemetry Collector for data collection and processing. Filtering is accomplished via appropriate Otel processors.
```
We should link to filter processor and transform processor here as well. I think mentioning pipelines and the metadata layer here is unnecessary, users shouldn't care about that for most of their filtering needs.